### PR TITLE
Fix qt5.15 deprecation warnings

### PR DESCRIPTION
**Pull request by omichel**
Qt 5.15 landed on MSYS2, happily breaking the Windows nightly builds because of deprecation warnings.
This PR fixes these warnings and hopefully the Windows nightly builds.

### CODE REVIEW

**Comment by omichel** [comment]
✔️ [appveyor tag build](https://ci.appveyor.com/project/omichel/webots-9ufow)

**Comment by DavidMansolino** [comment]
:eyes:

File: src/webots/sound/WbWaveFile.cpp [hunk 1]
```diff
@@ -211,20 +211,24 @@ void WbWaveFile::loadFromFile(int side) {
...
   static QString percentageChar = "%%";
 #endif
 
   QString outputFilename = WbStandardPaths::webotsTmpPath() + fi.baseName() + ".wav";
-  ffmpeg += " -y -i " + mFilename + " " + outputFilename + " -sample_fmt s16 -loglevel quiet";
+  QStringList arguments("-y");
+  arguments << "-i" << mFilename << outputFilename << "-sample_fmt"
+            << "s16"
+            << "-loglevel"
+            << "quiet";
```
Comment (DavidMansolino) [inline src/webots/sound/WbWaveFile.cpp:227]: Something like this (not tested) will allows us to make the arguments a const (which I slightly prefer, but this is personal).

```suggestion
  const QStringList arguments = QStringList() << "-y" << "-i" << mFilename << outputFilename << "-sample_fmt" << "s16"  << "-loglevel" << "quiet";
```

File: src/webots/control/WbControlledWorld.cpp [hunk 2]
```diff
@@ -71,7 +71,7 @@ WbControlledWorld::WbControlledWorld(WbProtoList *protos, WbTokenizer *tokenizer
   QFile file(WbStandardPaths::webotsTmpPath() + "WEBOTS_SERVER");
   if (file.open(QIODevice::WriteOnly)) {
     QTextStream stream(&file);
-    stream << mServer->fullServerName().toUtf8() << endl;
+    stream << mServer->fullServerName().toUtf8() << '\n';
```
Comment (DavidMansolino) [inline src/webots/control/WbControlledWorld.cpp:74]: Note that at the other occurrence of this issue you fixed it that way:
```suggestion
    stream << mServer->fullServerName().toUtf8() << Qt::endl;;
```

**Comment by stefaniapedrazzi** [comment]
We could also upgrade to Qt5.15 on Linux and Mac.
Or is there a reason to use anolder version of Qt on Mac?

File: src/webots/control/WbControlledWorld.cpp (hunk 2 above)
Comment (omichel) [reply src/webots/control/WbControlledWorld.cpp:74]: I had problems with compiling that on Linux. So, I believe it's simpler to use `\n` everywhere.

**Comment by DavidMansolino** [comment]
> We could also upgrade to Qt5.15 on Linux and Mac.
Or is there a reason to use an older version of Qt on Mac?

There is a rush for updating on Windows but not on the other platforms, but indeed it should be possible to update on the other platforms too.

**Comment by stefaniapedrazzi** [comment]
personally I think it would be better to try to upload Qt all the platforms rather than adding special cases for Windows.

**Comment by omichel** [comment]
I agree with you @stefaniapedrazzi, but the problem here is mainly to fix the Appveyor nightly builds which are broken. I propose to upgrade to Qt-5.15 on Linux and mac OS in another PR.

**Comment by omichel** [comment]
I restarted the travis builds: https://travis-ci.com/github/cyberbotics/webots/builds/168538941

**Comment by omichel** [comment]
It seems to compile now on Linux as well.
👍

File: src/webots/gui/WbStreamingServer.cpp [hunk 3]
```diff
@@ -48,6 +48,13 @@
 #include <QtWebSockets/QWebSocket>
 #include <QtWebSockets/QWebSocketServer>
 
+#include <QtCore/QDir>
+#include <QtCore/QFileInfo>
+#include <QtCore/QProcess>
+#include <QtCore/QString>
+#include <QtCore/QStringList>
+#include <QtCore/QTextStream>
```
Comment (DavidMansolino) [inline src/webots/gui/WbStreamingServer.cpp:56]: Are all these includes really needed?

Comment (omichel) [reply src/webots/gui/WbStreamingServer.cpp]: No, my mistake... I am removing them.

**Pull request merged**